File: packages/python/dynamic/runners/examples/callable.py
from dynamic.runners.callable import CallableRunner, CallableRunnerConfig

if __name__ == "__main__":
    print("Testing class...")

    def hello(msg):
        return msg
    
    config = CallableRunnerConfig(params=dict(msg="Hello World!\n-from Runner"))

    runner = CallableRunner(hello, config)

    print("Runner created and running...")
    print(runner.run())

File: packages/js/dynamic/src/runners/examples/callable.ts
import { Config } from '../types';
import { CallableRunner } from '..';

const add = ({ a, b }: { a: number; b: number }): number => a + b;

const config: Config = {
  input: {
    a: 1,
    b: 2,
  },
};

const runner = new CallableRunner(add, config);

console.log('runner.run(): ', runner.run());
